web/client/themes/default/less/react-data-grid.less

```diff
@@ -301,3 +301,12 @@
          }
      }
 }
+#page-rulesmanager{
```

Comment:
where this id is defined?

Reply:
it is concatenating ---> page-"id"
![image](https://github.com/geosolutions-it/MapStore2/assets/58145645/4bb4aaa1-d4f8-4b15-8aa7-8180d8e60224)

then here:
![image](https://github.com/geosolutions-it/MapStore2/assets/58145645/09a8e6d9-77c0-438b-b4ee-a25dd85ba29e)